unittests/test.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<espa_metadata version="1.3"
-xmlns="http://espa.cr.usgs.gov/v1"
+<espa_metadata version="2.0"
```

Comment:
Not for the data files themselves.  But It should be possible for the Metadata class to be more flexible.  I'll write a Jira ticket to someday address that.